github-runner-manager/tests/unit/test_config.py

```diff
@@ -170,13 +201,14 @@ def test_configuration_roundtrip(app_config: ApplicationConfiguration):
     assert app_config == reloaded_config
 
 
-def test_load_configuration_from_yaml(app_config: ApplicationConfiguration):
+def test_load_configuration_from_yaml(app_config: ApplicationConfiguration, openstack_config):
```

Comment:
nit: missing typehint
```suggestion
def test_load_configuration_from_yaml(app_config: ApplicationConfiguration, openstack_config: OpenStackConfiguration):
```

Reply:
Removed the code due to changes.